src/CoreBundle/Resources/views/snippets/common/download/download.html.twig

```diff
@@ -0,0 +1,22 @@
+{# @var downloadLinkDataModel \ChameleonSystem\CoreBundle\DataModel\DownloadLink #}
+
+{% if downloadLinkDataModel.backendLink %}
+    {% set internalAtribute -%}
```

Comment:
Is this set here to no clutter the block below?

If the id is the only intended content here I would call it like this. ie `downloadId` (or `downloadTypeId`)?

Reply:
done